charts/rqlite/README.md

```diff
@@ -0,0 +1,115 @@
+# rqlite Helm Chart
+
+[rqlite](https://rqlite.io) is a lightweight, easy-to-use, distributed relational database
+built on SQLite.
+
+Experienced Helm user?  Let's cut to the chase: the chart's default
+[`values.yaml`](values.yaml) is what you want.
+
+
+## Quick Start
+
+First add the `rqlite` helm repository:
+
+```bash
+helm repo add rqlite https://rqlite.github.io/helm-charts
+```
+
+To install or upgrade a release named `rqlite` in a namespace called `db` run:
+
+```bash
+helm upgrade -i -n db --create-namespace rqlite rqlite/rqlite
+```
+
+This uses the default chart values, which deploys a single-node rqlite cluster on 10GiB
+persistent volumes, without any authentication or TLS, accessible from within the
+Kubernetes cluster at `http://rqlite.db.svc.cluster.local`.
+
+If you want a 3 node cluster, add `--set replicaCount=3` at the end of the command.  Or,
+if you'd like to change the storage size to 50GiB, say, add `--set
+persistence.size=50Gi`.
+
+Naturally, once you have more than handful of customizations you will want to use a
+separate values file.  For example, `values.yaml` may contain:
+
+```yaml
+replicaCount: 3
+persistence:
+  size: 50Gi
+resources:
+  requests:
+    cpu: 500m
+```
+
+Then you deploy with:
+
+```bash
+helm upgrade -i -n db --create-namespace rqlite rqlite/rqlite -f values.yaml
+```
+
+Refer to [Helm's documentation](https://helm.sh/docs/) for more usage details for Helm itself.
+
+Finally, read through the chart's default [`values.yaml`](values.yaml), which is well
+commented and currently acts as the authoritative source of the chart's configuration
+values.
+
+
+## Production Deployments
+
+The default chart values will deploy a complete unsecured rqlite cluster, which is geared
+toward low-friction testing. But it means that anyone with network access to the K8s
+Service or pods has free rein over the rqlite database.
+
+A proper production deployment should define:
```

Comment:
I'm not sure I completely agree with this statement. A proper production deployment could be perfectly acceptable without all the security you mention below. Operators may be running in an environment where security is provided in a different manner. So I think the need for, say, password-based auth, is dependent on a specific situation. I would not like people to think they *need* these specific security options. I would prefer they *think carefully* about what it is they actually need. Open to hearing your opinion of course.

Good to warn, at least.

Reply:
Well fair enough.  I tend to have rather specific opinions about what a production deployment should look like which is informed by my own professional experiences.  You're right that it may be too opinionated in a broader context.  I will soften the language here.